traffic_ops/app/lib/Schema/Result/Deliveryservice.pm

```diff
@@ -138,7 +138,7 @@ __PACKAGE__->table("deliveryservice");
 =head2 max_dns_answers
 
   data_type: 'bigint'
-  default_value: 0
+  default_value: 5
```

Comment:
Did you mean to be changing the max dns answers here and below?

Reply:
I'm not intentionally changing it as part of this PR, but the generated schema files currently seem to be out of date for this field. After generating the new schema result, I didn't go through and clean up any of the other changes it made to this file. 

Do you think this change is incorrect?

Reply:
Not necessarily incorrect, but didn't seem to be correct for this particular PR.